test/ggsn_SUITE.erl

```diff
@@ -354,6 +356,18 @@ create_pdp_context_request_invalid_apn(Config) ->
     meck_validate(Config),
     ok.
 
+%%--------------------------------------------------------------------
+create_pdp_context_request_dotted_apn() ->
+    [{doc, "Check dotted APN return on Create PDP Context Request"}].
+create_pdp_context_request_dotted_apn(Config) ->
+    S = make_gtp_socket(Config),
+
+    create_pdp_context(dotted_apn, S),
```

Comment:
I think there is a mistake with cherry-picking it from `stable/1.x` because testing helpers are bit different. The code should look like:

```erlang
create_pdp_context_request_dotted_apn(Config) ->
    create_pdp_context(dotted_apn, Config),
    ...
```